src/main/java/pl/tomaszdziurko/jvm_bloggers/mailing/BlogSummaryMailSender.java

```diff
@@ -36,11 +34,13 @@ public BlogSummaryMailSender(BlogSummaryMailGenerator blogSummaryMailGenerator,
                                  MailSender sendGridMailSender,
                                  MailingAddressRepository mailingAddressRepository,
                                  IssueNumberRetriever issueNumberRetriever,
+                                 MailingSleepIntervalProvider mailingSleepIntervalProvider,
                                  NowProvider nowProvider) {
```

Comment:
This fat, big and ugly handcrafted constructor can be replaced by `@RequiredArgsConstructor(onConstructor = @__(@Autowired))` class level annotation. This way we can get rid of 13 lines of boilerplate code.